ocs_ci/utility/prometheus.py

```diff
@@ -13,6 +13,53 @@
 logger = logging.getLogger(name=__file__)
 
 
+def check_alert_list(
+    label,
+    msg,
+    alerts,
+    states,
+    severity="warning"
+):
+    """
+    Check list of alerts that there are 2 alerts with requested label and
+    message. First alerts should be pending and the second alert should be
+    firing.
+
+    Args:
+        label (str): Alert label.
```

Comment:
Please delete the points at the end of the sentences (to keep the convention)

Reply:
Done